hack/verify-container-images.sh

```diff
@@ -22,48 +22,25 @@ if [[ "${TRACE-0}" == "1" ]]; then
     set -o xtrace
 fi
 
-TRIVY_VERSION=0.34.0
+VERSION=${1}
 
```

Comment:
@killianmuldoon Found in https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/pull/2299/files. We probably still want to set GOARCH